### ISSUE TITLE
"Failed to connect to port" and "no such session" errors

### ISSUE THREAD

**Issue by claytonrcarter** [duncan3dc/dusk#3]
Thanks for this package! I just got it setup and thrashed against a few obtuse errors for a few hours. 

At first, calling `new Dusk` wouldn't start the `ChromeDriver` and I would get the "Failed to connect to port 9515" message. If I started `chromedriver-mac` manually, that message went away.

After that, I started getting "no such session" errors when trying to `visit` pages with the `Dusk` instance. I eventually fixed this by simply opening a new tab w/i in the chrome window that popped up. (I did this manually.) For whatever reason, this seems to have ironed out whatever was causing **both** of the above errors and everything appears to be working as advertised now: calling `new Dusk` starts Chrome and then my automation scripts run just fine.

I don't know that there's anything you can do about this, but wanted to share my experience in case it helps someone else with similar issues.

**Comment by duncan3dc** [comment]
Hi @claytonrcarter, are you using version `0.3.1` or above? There was an issue with earlier versions that sounds like what you've described.

**Comment by claytonrcarter** [comment]
Yes, I've got `0.4.0` installed. It cropped up again yesterday and took me a few minutes to fiddle out. It seems as though something is keeping chromedriver from launching via PHP until I mess with one of the windows left behind. It appears that adding a new tab and closing all of the left behind Chrome instances does something that then lets chromedriver start. I notice an additional (to the one I started manually) instance of chromedriver running once is working smoothly. And FWIW, it appears that it isn't killed after the tests/scripts are run.

I'm on PHP `7.1.6`, duncan3dc/dusk `0.4.0`, laravel/dusk `1.1.0`
